src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Swift/SwiftTypes.cs

```diff
@@ -38,6 +38,42 @@ public SwiftSelf(void* value)
         public void* Value { get; }
     }
 
+    /// <summary>
+    /// Represents the Swift frozen struct T, which is either enregistered into multiple registers,
```

Comment:
Nit. Even though the name is explanatory by itself, maybe we could add a comment that `SwiftSelf<T>` represents the Swift `self` context (similarly as we have for regular `SwiftSelf`)? I feel like it might cause a confusion that `SwiftSelft<T>` should be use for all "Swift frozen struct T".
```suggestion
    /// Represents the Swift 'self' context when the argument is Swift frozen struct T, which is either enregistered into multiple registers,
```